### PR TITLE
chore(amplify-ui): fix docs to change imports to use new packages ui-react-geo and ui-react-notifications

### PR DESCRIPTION
**Pull request by sreeramsama**
#### Description of changes:
Fix docs to change imports to use new packages `ui-react-geo` and `ui-react-notifications`

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [x] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [x] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [x] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by 0618 — DISMISSED** [review]

**Comment by calebpollman** [comment]
Maybe I'm missing something but not seeing any In-App example install scripts updated to include _@aws-amplify/ui-react-notifications_. Can we make sure we have audited/manually gone through any In-App related getting started guides to ensure we have not missed anything?

**Comment by sreeramsama** [comment]
> Maybe I'm missing something but not seeing any In-App example install scripts updated to include _@aws-amplify/ui-react-notifications_. Can we make sure we have audited/manually gone through any In-App related getting started guides to ensure we have not missed anything?

All install scripts and code snippets have already been updated in this PR. Please review changes in folder `src/fragments/lib/in-app-messaging/`.

File: src/fragments/lib/geo/js/geofences.mdx [hunk 1]
```diff
@@ -14,7 +14,12 @@ To add a geofence management component to your map, you can use the [amplify-geo
 Install the necessary dependencies with the following command:
 
 ```bash
-npm install maplibre-gl maplibre-gl-js-amplify aws-amplify @aws-amplify/ui-react
+npm install maplibre-gl maplibre-gl-js-amplify aws-amplify @aws-amplify/ui-react-geo
+```
+
+To use `withAuthenticator` as in the example below, install the necessary dependency with the following command:
```
Comment (calebpollman) [inline src/fragments/lib/geo/js/geofences.mdx:20]: If the example uses `withAuthenticator`, then we should include _@aws-amplify/ui-react_ in the above install script right?

**Comment by abdallahshaban557** [comment]
LGTM - But will prefer to have @hdworld11 review as well.

File: src/fragments/lib/in-app-messaging/integrate-your-application/js/install-dependencies.mdx [hunk 2]
```diff
@@ -19,5 +13,5 @@ import js0 from '/src/fragments/lib/in-app-messaging/integrate-your-application/
 </Callout>
 
 ```bash
-npm install @aws-amplify/ui-react
+npm install aws-amplify @aws-amplify/ui-react-notifications
```
Comment (calebpollman) [inline src/fragments/lib/in-app-messaging/integrate-your-application/js/install-dependencies.mdx:16]: Just to confirm, do we reference _@aws-amplify/ui-react_ anywhere in the example code still? Seems likely there is still the style import, if so, believe we should include both packages

Comment (reesscot) [reply src/fragments/lib/in-app-messaging/integrate-your-application/js/install-dependencies.mdx:16]: +1

File: src/fragments/lib/geo/js/geofences.mdx (hunk 1 above)
Comment (sreeramsama) [reply src/fragments/lib/geo/js/geofences.mdx:20]: Yes, makes sense to add it in the same line above as we include all dependencies used in the example.

File: src/fragments/lib/in-app-messaging/integrate-your-application/js/install-dependencies.mdx (hunk 2 above)
Comment (sreeramsama) [reply src/fragments/lib/in-app-messaging/integrate-your-application/js/install-dependencies.mdx:16]: Yes, we use styles from `@aws-amplify/ui-react`. Will update this install script.

File: src/fragments/lib/geo/js/geofences.mdx [hunk 3]
```diff
@@ -14,7 +14,7 @@ To add a geofence management component to your map, you can use the [amplify-geo
 Install the necessary dependencies with the following command:
 
 ```bash
-npm install maplibre-gl maplibre-gl-js-amplify aws-amplify @aws-amplify/ui-react
+npm install maplibre-gl maplibre-gl-js-amplify aws-amplify @aws-amplify/ui-react @aws-amplify/ui-react-geo
```
Comment (calebpollman) [inline src/fragments/lib/geo/js/geofences.mdx:17]: Let's move the external 3rd party dependencies to the end of the install script:
```suggestion
npm install aws-amplify @aws-amplify/ui-react @aws-amplify/ui-react-geo maplibre-gl-js-amplify maplibre-gl
```

File: src/fragments/lib/in-app-messaging/integrate-your-application/js/hoc-setup.mdx [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
+import { withInAppMessaging } from '@aws-amplify/ui-react-notifications';
 
 import '@aws-amplify/ui-react/styles.css';
 
-const App = () => (
-  {/* Your application */}
-);
+const App = () => ({
+  /* Your application */
+});
```
Comment (calebpollman) [inline src/fragments/lib/in-app-messaging/integrate-your-application/js/hoc-setup.mdx:10]: Is this intentional? Returning an object from a React component is invalid

Comment (sreeramsama) [reply src/fragments/lib/in-app-messaging/integrate-your-application/js/hoc-setup.mdx:10]: This section describe how users can wrap their App with `withInAppMessaging`.
https://pr-5508.d39agkzf0cwtdz.amplifyapp.com/lib/in-app-messaging/integrate-your-application/q/platform/js/#integrate-in-app-messaging
The next section has a more detailed example of the content inside App component.

**Review by reesscot — APPROVED** [review]

File: src/fragments/lib/in-app-messaging/integrate-your-application/js/hoc-setup.mdx (hunk 4 above)
Comment (sreeramsama) [reply src/fragments/lib/in-app-messaging/integrate-your-application/js/hoc-setup.mdx]: Updated to
```
const App = () => (
  <>In-App Messaging</>
);

```

**Review by reesscot — APPROVED** [review]

**Review by calebpollman — APPROVED** [review]

**Review by hdworld11 — APPROVED** [review]

**Review by thaddmt — APPROVED** [review]
LGTM

**Review by stocaaro — APPROVED** [review]

**Review by AllanZhengYP — APPROVED** [review]

**Review by timngyn — APPROVED** [review]

**Pull request merged**